static/styles/compose.css

```diff
@@ -277,7 +277,26 @@
     display: none;
 }
 
-.compose_banner {
+.above_composebox_banner {
+    background-color: hsl(147, 43%, 92%);
+    border: 1px solid hsla(147, 57%, 25%, 0.4);
+    color: hsl(147, 57%, 25%);
+
+    .above_composebox_banner_close_button {
+        color: hsla(147, 57%, 25%, 0.5);
+
+        &:hover {
+            color: hsl(147, 57%, 25%);
+        }
+
+        &:active {
+            color: hsla(147, 57%, 25%, 0.75);
+        }
+    }
+}
+
+.compose_banner,
+.above_composebox_banner {
```

Comment:
The separate + shared styling feels a bit clunky to me, but the banner has a different classname. 

Another option is to have them used a shared classname. So this new banner would be the success styling. The banner would have some more generic classname that applies to both kinds of banners (TBD).